crates/networking/p2p/rlpx/message.rs

```diff
@@ -50,6 +66,8 @@ impl Display for Message {
             Message::Ping(_) => "p2p:Ping".fmt(f),
             Message::Pong(_) => "p2p:Pong".fmt(f),
             Message::Status(_) => "eth:Status".fmt(f),
+            Message::GetAccountRange(_) => "snap::GetAccountRange".fmt(f),
```

Comment:
single colons (`:`) to be homogeneous with the rest of the representations.

Reply:
Nice catch!